Makefile

```diff
@@ -1,15 +1,15 @@
-.PHONY : develop build clean clean_pyc doc lint-diff black
+.PHONY : develop build clean clean_pyc doc lint-diff black check
 
 all: develop
 
 clean:
-	-python setup.py clean
+	-python3 setup.py clean
```

Comment:
What is the reason for having ``python3`` rather than ``python``?
When using virtual environment, it will be just ``python``.
When working on windows (unluckily) there is ``python`` or ``py`` or ``py -3``, but no ``python3``.

Reply:
when not using a virtualenv on OSX or Linux, `python` still points at python2

Reply:
Thus, it will break make on windows.
What about setting $(PYTHON) variable depending on OS?
https://stackoverflow.com/a/14777895

Reply:
Yea I'd rather just keep these as is. I'm pretty sure macOS and most linux distros have or are in the process of removing the standard py2 installation, so this is adding what will soon be legacy code style

Reply:
Does make even work on windows?

Reply:
It is possible to use make on windows. I use it. in GitBash terminal (MinGW) it works almost as in bash.

Reply:
deal breaker?  i think osx/linux use case is more common, but not a hill worth dying on.